plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -1,11 +1,10 @@
 pragma solidity 0.4.18;
+import 'SafeMath.sol';
```

Comment:
I think we should keep the newline between the `pragma` and `import` statements.